Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -309,6 +309,8 @@ public void run() {
                     ess.getLogger().log(Level.INFO, "Set god mode to false for {0} because they had it enabled without permission.", user.getName());
                 }
 
+                user.setPromptingClearConfirm(false);
```

Comment:
`setPromptingClearConfirm` overwrites the user's clear confirmation setting, which is what they manually choose with `/clearinventoryconfirmtoggle`. To reset the pending command, use `setConfirmingClearCommand`, which overwrites the temporary string representing the command they just ran.

Reply:
Ahh, I see. I must have misinterpreted the method. I should have read it more thoroughly.